opentracing-impl/src/main/java/io/opentracing/AbstractNoopSpan.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+final class AbstractNoopSpan extends AbstractSpan implements NoopSpan, NoopSpanContext {
+
+    static final AbstractNoopSpan INSTANCE = new AbstractNoopSpan("noop");
```

Comment:
Have renamed package to `no.opentracing.impl`
This allows the noop classes to keep simpler names.

This will cause compatibility headaches, but we're pre-1.0 version.